pvlib/test/test_pvsystem.py

```diff
@@ -266,6 +266,17 @@ def test_PVSystem_sapm_spectral_loss(sapm_module_params):
     out = system.sapm_spectral_loss(airmass)
 
 
+@pytest.mark.parametrize("expected", [1.03173953])
+    
```

Comment:
python convention is no blank lines between decorators and the function.

Reply:
Fixed